server/service/integration_mdm_test.go

```diff
@@ -5811,6 +5822,10 @@ func (s *integrationMDMTestSuite) TestBootstrapPackageStatus() {
 	s.DoJSON("GET", "/api/latest/fleet/mdm/bootstrap/summary", nil, http.StatusOK, &summaryResp)
 	require.Equal(t, fleet.MDMAppleBootstrapPackageSummary{Pending: uint(len(noTeamDevices))}, summaryResp.MDMAppleBootstrapPackageSummary)
 
+	var lhr listHostsResponse
+	s.DoJSON("GET", "/api/latest/fleet/hosts", nil, http.StatusOK, &lhr, "team_id", "0", "bootstrap_package", "pending")
+	require.Len(t, lhr.Hosts, len(noTeamDevices))
+
```

Comment:
nice! would it be possible to add a test for the endpoint with the summary as well?

Reply:
(maybe it's implicitly already covered?)

Reply:
@roperzh is it covered by the lines just above these? Looks like those hit the `/bootstrap/summary` endpoint

Reply:
nice, thanks for the sanity check!